container-builder/common/registry_docker.go

```diff
@@ -74,12 +77,12 @@ func (d DockerLocalRegistry) StartRegistry() string {
 	registryID := d.GetRegistryRunningID()
 
 	if len(registryID) > 0 {
-		logrus.Infof("Registry ID %s is already running", registryID)
+		klog.V(log.I).InfoS("Registry ID is already running", "ID", registryID)
```

Comment:
Why not use Infof ?

Reply:
@ricardozanini wanted to introduce structured logging.